src/main/java/uk/gov/companieshouse/api/testdata/service/RandomService.java

```diff
@@ -42,4 +42,6 @@
      * @return A base-64-encoded string
      */
     String addSaltAndEncode(String baseString, int saltLength);
+
+    String generateEntityId();
```

Comment:
Do we need this extra operation here? If we need the id to start with 8 I'd say that that is a specific requirement for a given service and does not really need to be included in a random service? It could also help with the unit test as we can ensure the id always start with 8 rather than relying on whatever the random service returns.